tests/multigrid/transfer_matrix_free_02_block.with_mpi=true.with_p4est=true.with_trilinos=true.mpirun=5.output

```diff
@@ -0,0 +1,259 @@
+
```

Comment:
Same here: I would delete this one and only keep the single processor case (or the other way around).